modules/apigee_m10n_teams/tests/src/Kernel/Entity/RatePlanEntityKernelTest.php

```diff
@@ -122,18 +122,16 @@ public function testRatePlanEntity() {
     $this->setRawContent((string) \Drupal::service('renderer')->renderRoot($build));
 
     // Rate plans as rendered.
-    static::assertSame($rate_plan->getDisplayName(), trim($this->cssSelect('.apigee-package-rate-plan > div > a')[0]));
```

Comment:
If the template is now showing the entity label in the template, we should probably test the `h2` instead of removing the assertions.